### DISCUSSION TITLE
downloadhelper - vimeo - não consegue fazer donwload

### DISCUSSION THREAD

**Discussion by EspacoConsciencia** [aclap-dev/video-downloadhelper#172] [Help and Q&A] [answered]
### Your question

Boa tarde,
Necessitávamos de ajuda, antes conseguíamos fazer download dos vídeos do vimeo, mas agora ou faz o download e o vídeo fica acelerado ou então o ficheiro é detetado mas só consegue fazer o donwload dos primeiros minutos.
Conseguem ajudar-me nesta questão?

### Addon & CoApp version and details

windows 10.
Video DownloadHelper 8.2.0.24 (stable)

**Comment by paulrouget** [comment, marked as answer]
I need an URL.

**Reply by EspacoConsciencia** [reply]
dos vídeos que queremos fazer donwload  - https://vimeo.com/463758519
ou do donwloadhelper - chrome-extension://lmjnegcaeklhafolokijcfjliaokphfk/content/settings.html?panel=settings

**Reply by paulrouget** [reply]
Please wait until the addon update to version 8.2.1 (it will auto-update). If it doesn't fix your issue, please tell me.

**Reply by EspacoConsciencia** [reply]
ok. muito obrigada =)

**Reply by EspacoConsciencia** [reply]
Olá, o update já aparece, mas infelizmente ainda continua igual =(
Envio uns prints no word em anexo, espero que ajude. 
Obrigada
[downloadhelper.docx](https://github.com/aclap-dev/video-downloadhelper/files/14363674/downloadhelper.docx)

**Reply by paulrouget** [reply]
Understood. I'll look into it.

**Reply by EspacoConsciencia** [reply]
Muito obrigada pela atenção e ajuda, o que puder ajudar diga-me por favor.
Não percebo muito por isso pedi ajuda.

**Reply by Mattmjs** [reply]
In your attachment , no download has been started. Are you sure you clicked on one of the down pointed arrows ?
Otherwise what is the issue ? 

![Screenshot 2024-02-23 173051](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/95a97b07-7332-40c6-8153-7fccd247b9de)

**Reply by EspacoConsciencia** [reply]
Sim carreguei sim, o problema é que não faz donwload do tempo total do vídeo, o vídeo tem cerca de 2horas e tal e ele só faz donwload dos primeiros 9 minutos e 29 segundos.

**Reply by Mattmjs** [reply]
Oh yes, it is a display problem in the extension. Whenever the duration is over an hour, the time is shown incorrectly.
For example yours is 2:09:21 , but instead it is displayed as 9:21.

But be assured that if you launched the download you'll get whole 2 hour 9 minute video. Try it out and come back with the result.

**Reply by EspacoConsciencia** [reply]
ok, sim já consegui fazer o download, mas antes aparecia o tempo todo mesmo sendo vídeos com duração de mais de uma hora, e mesmo assim cheguei a tentar e não conseguia fazer o download todo. Mas agora já consegui =) Muito obrigada pela ajuda, foram incríveis, obrigada =)

**Comment by Mattmjs** [comment]
What was the issue on this one ? , I see a 2 hour video in the attachment  but no download has been started.
And it's not public to view.

No mention of companion version either

**Reply by paulrouget** [reply]
No idea. I haven't tested private Vimeo much. You have an experience with that?

**Reply by Mattmjs** [reply]
Not a lot with private vimeo video, there were quite a few examples on the google group but very few were private.
Although I think it is mostly resolved now. You fixed dash streaming videos that use **& range=** in the streaming media.

There is still the issue of the other type using chop segment streaming, these seem to finish 4 5 or 6 seconds  short of the full
duration.

**Reply by paulrouget** [reply]
Yeah I saw that one. I haven't investigated much.

**Reply by Mattmjs** [reply]
Is it public or private video , share the video link here if not private.

**Reply by EspacoConsciencia** [reply]
This a private video. sorry!
